modules/rosa-rhoda.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * adding_service_cluster/rosa-available-services.adoc
+
+[id="rosa-rhoda_{context}"]
+= Red Hat OpenShift Database Access
+
+Red Hat OpenShift Database Access enables easy consumption of database-as-a-service (DBaaS) offerings from partners including MongoDB Atlas, Crunchy Bridge, and CockroachDB directly from managed OpenShift clusters. You can manage, monitor, and create cloud-hosted database instances for connecting to your applications.
```

Comment:
I don't know if OSD/ROSA have different rules, but for OCP we usually don't write out "OpenShift" by itself like this here in "managed OpenShift". Usually we write {product-title}, but I'm not sure if that's right for this situation. Could you maybe check with the other managed writers to see what they would do?

Reply:
Attributes were added to match best practices in Docs.